app/dashboard/views.py

```diff
@@ -4499,13 +4546,18 @@ def join_tribe(request, handle):
     if request.user.is_authenticated:
         profile = request.user.profile if hasattr(request.user, 'profile') else None
         try:
-            TribeMember.objects.get(profile=profile, org__handle=handle.lower()).delete()
+            try:
+                TribeMember.objects.get(profile=profile, org__handle=handle.lower()).delete()
+            except TribeMember.MultipleObjectsReturned:
+                TribeMember.objects.filter(profile=profile, org__handle=handle.lower()).delete()
+
```

Comment:
should be situations where profile return multiple? and in that case do we want to delete all of them?

Reply:
shouldn't ever be multiple members for a single profile and an org.

Reply:
the issue is that if a request fails, the user could get multiple entries in the db and then can't follow/unfollow properly, this ensures that we delete any multiple entries and keep the user's ability to follow/unfollow in tact.